DatadogSessionReplay/Sources/SessionReplay+objc.swift

```diff
@@ -107,4 +115,31 @@ public enum DDSessionReplayConfigurationPrivacyLevel: Int {
     }
 }
 
+/// Available image privacy levels for image masking.
+@objc
+public enum DDImagePrivacyLevel: Int {
+    /// Only SF Symbols and images loaded using UIImage(named:) that are bundled within the application package will be recorded.
+    case maskNonBundledImages
+    /// No images will be recorded.
+    case maskAll
+    /// All images including the ones downloaded from the Internet during the app runtime will be recorded.
+    case maskNone
+
+    internal var _swift: ImagePrivacyLevel {
+        switch self {
+        case .maskNonBundledImages: return .maskNonBundledImages
```

Comment:
Do we not prefer maskNonBundledOnly for this? This would align with android (MASK_LARGE_ONLY), and also as an ImagePrivacyLevel the word "images" here could be superfluous